app/controllers/index.js

```diff
@@ -17,22 +17,24 @@ export default class IndexController extends Controller {
     let users = await all(fetchRequests);
     users = this.identifyUsers(users);
     users = this.sortUsers(users);
+    users = await this.fetchAdditionalUserDetails(users);
```

Comment:
**thought:**

I think we could switch the order of `sortUsers` and `fetchAdditionalUserDetails`, so that we have the option to sort in a different way once we know additional information about users.

I think, for now, it's perfectly fine to leave this line as is.

Reply:
Oh yeah, that makes sense @ijlee2! But, I see there could be a catch where if a user's name is empty, what should we do on that data. Probably I'd like to take this in a separate PR.

Reply:
Yeah, I can see the preference to keep the names as given.

I was thinking we could remove Ember Data some time, heh. We only use 1 model, `github-pull`, for reading data. I think we could use `fetch` instead and simplify the app.

Reply:
Yep, agreed